var/spack/repos/builtin/packages/szx/package.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Szx(AutotoolsPackage):
+    """An ultra fast error bounded compressor for scientific datasets"""
+
+    homepage = "https://github.com/szcompressor/szx"
+    url = "https://github.com/szcompressor/szx"
+    git = "ssh://git@github.com/szcompressor/szx"
```

Comment:
Note you will generally be prompted for password etc. unless you have set up passwordless ssh authentication with github (FYI if you have, other users would likely be prompted for credentials): I recommend an `https`-based URL

Reply:
This package was private when this package was created.  I'll update the URL if this becomes public.

Reply:
To be clear: I'm OK waiting for that, but I wouldn't want to merge the PR until either:

* The repo becomes public (and the `git` property is changed) or...
* This package is removed

It would be problematic to add packages which other users cannot use.

Reply:
@scheibelp of course. Based on @xiaodong-yu response above, I'll push a commit that removes this until we we get approval for release.

Reply:
should be gone now.